app/views/hosts/console/spice.html.erb

```diff
@@ -2,7 +2,7 @@
 <% title "#{@console[:name]}" %>
 <%= title_actions(
         button_group(link_to(_("Ctrl-Alt-Del"), "#", :id => "sendCtrlAltDelButton", :onclick => 'sendCtrlAltDel()', :class => "btn btn-default"),
-                     link_to(_("Troubleshooting"), "http://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#7.1NoVNC",
```

Comment:
the issue also suggests to rename "troubleshooting" to"documentation", which would be consistent with other places